Jenkinsfile

```diff
@@ -42,6 +42,14 @@ library identifier: "ci-pipeline@${env.ghprbActualCommit}",
                                        [$class: 'RefSpecsSCMSourceTrait',
                                         templates: [[value: '+refs/heads/*:refs/remotes/@{remote}/*'],
                                                     [value: '+refs/pull/*:refs/remotes/origin/pr/*']]]]])
+library changelog: false,
+        identifier: 'multiarch-ci-libraries@dev',
+        retriever: legacySCM([$class: 'GitSCM', branches: [[name: '*/dev']],
+                              doGenerateSubmoduleConfigurations: false,
+                              extensions: [], submoduleCfg: [],
+                              userRemoteConfigs:
+                                      [[url: 'https://github.com/scoheb/multiarch-ci-libraries']]])
```

Comment:
Neat picking but that repo does not seem to have a clear licensing which makes the code there proprietary.

Reply:
Thanks @pypingou 
I added a license to the repo.

Reply:
Thanks :)